src/node/rpc/member_frontend.h

```diff
@@ -1804,6 +1866,56 @@ namespace ccf
       make_endpoint("create", HTTP_POST, json_adapter(create))
         .set_require_client_identity(false)
         .install();
+
+      // Only called from node. See node_state.h.
+      auto refresh_jwt_keys = [this](
+                                EndpointContext& args, nlohmann::json&& body) {
+        // All errors are server errors since the client is the server.
```

Comment:
Here, we should check that `args.rpc_ctx.session.caller_cert` is a `TRUSTED` node in the nodes table.

Reply:
Can I also check whether the request is coming from the same node?

Reply:
Let's defer this until #1904 is done. It's more tricky than it seems.